torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -218,11 +219,12 @@ class DeadCodeEliminator {
       }
     }
 
-    if (aliasDb_) {
-      if (aliasDb_->writesToAlias(node, liveValues_)) {
+    if (useAliasDb_) {
```

Comment:
In the old code, `aliasDb_` is always initialized, so I don't understand why the old code needs to check `if(aliasDb_)`, perhaps an unnecessary checking? If so, then we can remove `useAliasDb_` altogether.

Reply:
There are two constructors for DeadCodeEliminator, one initializes aliasDb_ and the other doesn't. Not sure why, but that's why I decided to use useAliasDb_ to capture that.